.changelog

```diff
@@ -22,3 +22,5 @@
   clones in Client Configuration Guide (bsc#1179277)
 - Adds note about Appstream Packages in Channels section
   of Client Cfg Guide (bsc#1179525)
+- Removes Public Cloud content from Install Guide, and updates
```

Comment:
This is wrong.

The Public Cloud QuickStart Guide is not a Public Cloud Guide. It is meant to give users a quick start, not to replace the Installation Guide. QuickStart Guides even use a different, more informal and direct language.